app/views/layouts/application.html.erb

```diff
@@ -20,6 +20,12 @@
     };
   </script>
 
+  <% if session.delete(:should_reset_jwt) %>
+    <script>
+      localStorage.setItem("jwt", null);
```

Comment:
Any reason to use this over `deleteItem`?

Reply:
It was because I was looking for `deleteItem` in my IDE, but didn't find it. That's because it's `removeItem`